portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,25 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+  console.log("Helloooo");
 }
+
+function randomImage(){
+    const img_container = document.getElementById('img-container');
```

Comment:
"Local variable names are written in lowerCamelCase" - https://google.github.io/styleguide/jsguide.html#naming-local-variable-names